test/fixtures/user-integration-app/server/server.js

```diff
@@ -0,0 +1,10 @@
+var loopback = require('../../../../index');
+var boot = require('loopback-boot');
+var app = module.exports = loopback();
+app.enableAuth();
+boot(app, __dirname);
+app.use(loopback.token({model: app.models.AccessToken}));
+var apiPath = '/api';
+app.use(apiPath, loopback.rest());
+app.use(loopback.urlNotFound());
+app.use(loopback.errorHandler());
```

Comment:
There are more models such as `post` and `blog` are needed. I'll merge the PR as is and defer the cleanup of tests.